docs/next.config.js

```diff
@@ -135,18 +135,6 @@ module.exports = {
                         '@mui/utils': '../packages/mui-utils/src',
                         '@mui/core': '../packages/mui-core/src',
                         '@mui/material-next': '../packages/mui-material-next/src',
-                        // all legacy package names in this monorepo
-                        '@material-ui/core': '../packages/mui-material/src',
-                        '@material-ui/docs': '../packages/mui-docs/src',
-                        '@material-ui/icons': '../packages/mui-icons-material/lib',
-                        '@material-ui/lab': '../packages/mui-lab/src',
-                        '@material-ui/styled-engine': '../packages/mui-styled-engine/src',
-                        '@material-ui/styles': '../packages/mui-styles/src',
-                        '@material-ui/system': '../packages/mui-system/src',
-                        '@material-ui/private-theming': '../packages/mui-private-theming/src',
-                        '@material-ui/utils': '../packages/mui-utils/src',
-                        '@material-ui/unstyled': '../packages/mui-core/src',
-                        '@material-ui/core/*': '../packages/mui-material/src/*',
```

Comment:
Was I right in assuming this was only needed in `notistack`?

Reply:
I think so.